subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/fixture/TestProjectLocator.groovy

```diff
@@ -26,7 +26,7 @@ class TestProjectLocator {
         if (!dir.directory) {
             throw new IllegalArgumentException("Did not find test project at: '$dir.absolutePath'. Please run 'gradlew $name' to generate the test project.")
         }
-        if (!new File(dir, "settings.gradle").file && !new File(dir, "settings.gradle.kts").file) {
+        if (!new File(dir, "settings.gradle").file && !new File(dir, "settings.gradle.kts").file && !new File(dir, "settings.gradle.something").file) {
```

Comment:
💅 This could use `GradleDsl.XXX.filenameFor("settings")` instead so it is easier to find it later